internal/test/context/context.go

```diff
@@ -18,8 +18,7 @@ func PrevSignatureMattersOnContext(t *testing.T, ctx context.Context) (context.C
 	require.NoError(t, err)
 
 	prevMatters := true
```

Comment:
should really be `prevMatters := sch.Name == crypto.DefaultSchemeID`, but that's a nitpick :p